crates/holochain/tests/tests/inline_zome_spec/mod.rs

```diff
@@ -354,7 +354,7 @@ async fn call_non_existing_zome_fails_gracefully() -> anyhow::Result<()> {
 
     let (alice,) = app.into_tuple();
 
-    // Call the a zome fn on a non existing zome on Alice's app
```

Comment:
This is a typo but it's a code comment rather than published in documentation, so I wouldn't accept this pull request just for that change.